### ISSUE TITLE
[Question]Batch-level moving average statistics in Batch Normalization do not exchange between GPUs

### ISSUE THREAD

**Issue by straywarrior** [NVIDIA-Merlin/HugeCTR#181]
In `BatchNormLayer`, the batch-level moving average statistics `result_save_mean_` and `result_save_inv_var_` are created, stored, and updated in each GPU and never exchange and between GPUs. So different GPUs may have different moving average statistics that are different from global statistics. Although this may have no effect on the final result if the data set is large enough and fully shuffled (so the data distribution is uniform). But is this behavior expected?

**Note**: Before filing an issue, you may want to check out [our compiled Q&A list](https://github.com/NVIDIA/HugeCTR/blob/master/docs/QAList.md) first.

**Comment by minseokl** [comment]
Yes it is the expected behavior. HugeCTR just doesn't support syncBatchNorm.
